client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -187,16 +193,12 @@ export const AppSchemaForm = ({ app }) => {
     initialValues.allocation = app.scheduler;
   }
 
-  // local state for alerts
-  const [visible, setVisible] = useState(true);
-  const onDismiss = () => setVisible(false);
-
   return (
     <div id="appForm-wrapper">
       {jobSubmission.response && (
         <div id="appForm-alerts">
           {jobSubmission.error ? (
-            <Alert color="warning" isOpen={visible} toggle={onDismiss}>
+            <SectionMessage type="warning" className="appDetail-error">
```

Comment:
No change required: In the future, should we add some bottom padding to `<SectionMessage>`? Or is that something to handle in its container.
<img width="728" alt="Screen Shot 2020-12-16 at 2 04 52 PM" src="https://user-images.githubusercontent.com/20326896/102400705-efc9cc00-3fa7-11eb-9850-ec154453ba4b.png">

Reply:
This `<SectionMessage>` has too much padding. This padding does not come from `<SectionMessage>`. Please remove what seems to be padding added to `.appDetail-error`.

I think @rstijerina is asking for bottom margin, which is something that should be added by some class (not from `<SectionMessage>`), probably `.appDetail-error`.

_I am not fully available to investigate this, but it should be resolved._

Reply:
After resisting this, I remember more about `appDetail-error`, there are more solution options:
- _(new possible solution)_
  **If** the `appDetail-error` is providing necessary styles, **then** ensure that the `<SectionMessage>` does **not** have that class.
- _([initial suggestion](https://github.com/TACC/Frontera-Portal/pull/228#discussion_r545229487))_
  **If** the `appDetail-error` is not providing necessary styles, **then** ensure that a `<SectionMessage class="appDetail-error">` does not have that padding.